.changelog/2268.txt

```diff
@@ -0,0 +1,8 @@
+```release-note:bug
+resource/cloud_backup_schedule: Handles resource deletion outside of Terraform
+resource/encryption_at_rest: Handles resource deletion outside of Terraform
+resource/push_based_log_export: Handles resource deletion outside of Terraform
+resource/search_deployment: Handles resource deletion outside of Terraform
+resource/stream_connection: Handles resource deletion outside of Terraform
+resource/stream_instance: Handles resource deletion outside of Terraform
```

Comment:
Suggest simplifying to "Deletes resource outside of Terraform".

Reply:
Was this what you meant? 19fe459